### DISCUSSION TITLE
Request for verification of academic email

### DISCUSSION THREAD

**Discussion by seheon6403** [RosettaCommons/rosetta#229] [ROSIE] [answered]
Hello,
I am a researcher at the Scripps Korea Antibody Institute in the Republic of Korea.
I am very interested in many of your technologies and would like to use them for academic purpose.
Therefore, I kindly request that you add our institute’s email domain to your list of verified academic email addresses.
The domain is skai.or.kr.

Thanks in advance

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! This should be fixed now. Please feel free to re-open this issues if you still having trouble logging-in.

**Reply by seheon6403** [reply]
Thank you for your prompt response.

However, the confirmation email for registration is not being sent. I keep clicking 're-send registration instructions,' but there is no response. Could you advise on how to resolve this issue?

**Reply by lyskov** [reply]
Hm… there should not be any confirmation emails since GitHub is handling all that… what address are you using to sign-in? Could you please try logging-in at https://r2.graylab.jhu.edu/auth/login and let me know if that works for you? Thanks,

**Reply by seheon6403** [reply]
I can log-in the website that you provide without any problem. Actually, I tried to log-in https://rosie.graylab.jhu.edu/ this website at first and I saw a notification message like the attached picture. Is the link you provided different from the website I initially logged into?
![제목 없음](https://github.com/user-attachments/assets/21c88f7c-9f05-41e9-b7ed-880ba0919065)

**Reply by lyskov** [reply]
Yes, these are different servers the https://rosie.graylab.jhu.edu is ROSIE-1 and it uses different account system. You should be able to self-register your account here: https://rosie.graylab.jhu.edu/signup
